libs/@blockprotocol/graph/src/internal/mutate-subgraph/edge.ts

```diff
@@ -43,3 +44,23 @@ export const addOutwardEdgeToSubgraphByMutation = <Temporal extends boolean>(
   }
   /* eslint-enable no-param-reassign */
 };
+
+/**
+ * @todo - with the introduction of non-primitive data types edges will need to be added here
+ *
+ * Looking to build a subgraph? You probably want {@link buildSubgraph} from `@blockprotocol/graph/stdlib`
+ *
+ * This MUTATES the given {@link Subgraph} by creating any ontology related edges that are **directly implied** by the given data type ids (see note below).
+ * Mutating a Subgraph is unsafe in most situations – you should know why you need to do it.
+ *
+ * *Note*: This only adds edges as implied by the given data types, if the {@link Subgraph} is invalid at the time of
+ * method call (e.g. by missing data type endpoints), this will not loop through the vertex set to finish incomplete
+ * edges.
```

Comment:
Is this correct? I thought we just don't create edges if they're missing. (I presume this comment applys to the docs of all of the new methods)

Reply:
Sorry I realise you could interpret it in a way that still makes sense.

I'd probably get rid of the whole sentence though and replace it with something else saying that this only creates edges where both endpoints are present within the `Subgraph`, regardless of if the Data Type references more vertices.

Reply:
Changed in 2696edc442a1d3898f77eb8caa3d7f4bb363f06a